src/utils/rootkeypackage_utils/src/rootkeypackage_utils.c

```diff
@@ -89,6 +90,490 @@ ADUC_Result ADUC_RootKeyPackageUtils_Parse(const char* jsonString, ADUC_RootKeyP
     return result;
 }
 
+/**
+ * @brief Helper function for comparing two constbuffers
+ *
+ * @param lConstBuff left constbuffer to compare
+ * @param rConstBuff right constbuffer to compare
+ * @return True if equal; false if not
+ */
+_Bool CompareConstBuffers(CONSTBUFFER_HANDLE lConstBuff, CONSTBUFFER_HANDLE rConstBuff)
```

Comment:
These comparison functions seem to be only used in test code, so moving it into a rootkey_testutils lib would be better.